index.js

```diff
@@ -673,7 +673,23 @@ class GithubScm extends Scm {
                 }
             });
 
-            return commit.data.object.sha;
+            if (refObj.data.object.type === 'tag') {
```

Comment:
You should add a comment like `// annotated tag` here and `// lightweight tag` to the next `else if` clause.

Reply:
Thank you. I have added these comments.